packages/core/src/system.ts

```diff
@@ -188,6 +189,9 @@ export function createSystem<T extends ActorSystemInfo>(
     get: (systemId) => {
       return keyedActors.get(systemId) as T['actors'][any];
     },
+    getRunningActors: () => {
```

Comment:
```suggestion
    getAll: () => {
```

Reply:
It should return all registered ones and we have to deregister them appropriately too - otherwise we'd have a memory leak.

Reply:
Ok. so it matches with what this PR indents for all actors. I'll rename 👍